### PR TITLE
[7152] Refactor lib.py functions (4-3 stable)

### PR DESCRIPTION
**Pull request by Nishant2022**
Rename user parameter to session and make it the first parameter

### CODE REVIEW

**Comment by Nishant2022** [comment]
Cherry pick from main

**Review by korydraughn — APPROVED** [review]
Looks identical to me.

Let's wait for one more approval before adding the pounds.

**Review by alanking — APPROVED** [review]
Cherry-picked from https://github.com/irods/irods/pull/7153

Please add #

**Review by alanking — APPROVED** [review]

**Pull request merged**
